libs/mongoose/Database.js

```diff
@@ -5,12 +5,19 @@ import PropertiesReader from 'properties-reader';
 const properties = PropertiesReader(path.resolve('voxnostra.properties'));
 const mongoDbProp = 'project.app.mongodb.devUrl';
 const mongooseUrl = process.env.MONGOB_URL || properties.get(mongoDbProp);
+
+let isFirst = true;
+
 const Database = async ({ req, res }) => {
-  if (!mongoose.connection) {
-    req.database = mongoose.connection;
-    return { req, res };
+  if (isFirst) {
+    await connect(mongooseUrl,
+      { useNewUrlParser: true, useCreateIndex: true, useUnifiedTopology: true });
+    isFirst = false;
+    // fixme: here we log connection errors and retries
```

Comment:
# Nitpick
Remove commented codes

Reply:
Most of what i commented are task that still need to be done and not related to my task, Like there is no logging infrastructure setup for the code base.

Once that task is done, i will add it to the code base, I should probably create an issue for that.